### DISCUSSION TITLE
Emergency Acces - Not all Items

### DISCUSSION THREAD

**Discussion by fabo123456** [lockerpm/.github#11] [Ideas]
Hello everyone

I would appreciate it if you don't have to automatically pass on all entries.
For example, I would like to entrust some entries to someone from my club. All other entries should be given to a family member.
So I would need two Emergency access with different Items.

**Comment by luongnh261** [comment]
Hey @fabo123456 , thanks for raising the request. We will add it to the backlog for consideration in future development :)
